netbox_metatype_importer/api/urls.py

```diff
@@ -1,10 +1,10 @@
-"""
```

Comment:
Issue: The code changes are missing the `basename` argument for the `ModuleTypeViewSet` router registration. 
Comment: The router registration for `ModuleTypeViewSet` is missing the `basename` argument, which should be provided to ensure proper routing.